requirements-data_and_models.txt

```diff
@@ -14,14 +14,8 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
-
```

Comment:
This line break shouldn't go away, isn't it?

Reply:
you want the empty line back?

Reply:
This is what we have in the other two requirements files. So, it will introduce an inconsistency.

Reply:
Fixed in 5f633d3